REST API is broken

AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the MembershipApplicationSerializer serializer.",)

--- a/project/access/rest.py
+++ b/project/access/rest.py
@@ -30,6 +30,7 @@ class TokenSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Token
+        fields = '__all__'
 
 
 class TokenFilter(filters.FilterSet):
@@ -55,6 +56,7 @@ class AccessTypeSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = AccessType
+        fields = '__all__'
 
 
 class AccessTypeFilter(filters.FilterSet):
@@ -78,6 +80,7 @@ class GrantSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Grant
+        fields = '__all__'
 
 
 class GrantFilter(filters.FilterSet):

--- a/project/creditor/rest.py
+++ b/project/creditor/rest.py
@@ -30,7 +30,7 @@ class TransactionSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Transaction
-
+        fields = '__all__'
 
 class TransactionFilter(filters.FilterSet):
 
@@ -56,6 +56,7 @@ class RecurringTransactionSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = RecurringTransaction
+        fields = '__all__'
 
 
 class RecurringTransactionFilter(filters.FilterSet):

--- a/project/members/rest.py
+++ b/project/members/rest.py
@@ -31,6 +31,7 @@ class MembershipApplicationTagSerializer(serializers.HyperlinkedModelSerializer)
 
     class Meta:
         model = MembershipApplicationTag
+        fields = '__all__'
 
 
 class MembershipApplicationTagFilter(filters.FilterSet):
@@ -83,6 +84,7 @@ class MembershipApplicationSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = MembershipApplication
+        fields = '__all__'
 
 
 class MembershipApplicationFilter(filters.FilterSet):

